src/services/ReportService.js

```diff
@@ -8,8 +10,107 @@ export default class ReportService {
    * @returns {Promise<string>} Nome do arquivo XLSX temporário.
    */
   async generateCompleteReport(institutionID) {
-    // TODO: implementar na task B03 (https://trello.com/c/VXgx1w4r)
-    // usar funções TechnologyService.exportTechnology(...), ExecutionService.exportExecutions(...), ExecutionService.exportConsolidatedExecutions(...)
-    // usar lib exceljs para gerar o arquivo XLSX (https://www.npmjs.com/package/exceljs)
+
+    const ExcelJS = require("exceljs")
+
+    //Cria arquivo e adiciona algumas propriedades
+    const workbook = new Excel.Workbook()
+    workbook.creator = "Me"
+    workbook.lastModifiedBy = "Her"
+    workbook.created = new Date(1985, 8, 30)
+    workbook.modified = new Date()
+    workbook.lastPrinted = new Date(2016, 9, 27)
```

Comment:
Tirar `modified` e `lastPrinted`, e ajustar as outras propriedades:

```suggestion
    workbook.creator = "dTool - AGES, PUCRS"
    workbook.lastModifiedBy = "dTool - AGES, PUCRS"
    workbook.created = new Date()
```